omniduct/databases/_schemas.py

```diff
@@ -188,7 +188,7 @@ def all(self):
         return self._table_names
 
     def __dir__(self):
-        return self.all()
+        return self.all
```

Comment:
only tested on snowflake

Reply:
Yeah... this was a bug. My bad. Thanks for fixing.